.github/workflows/release.yml

```diff
@@ -0,0 +1,168 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      publish:
+        description: "Publish to PyPI (true/false)"
+        required: true
+        default: "false"
+
+jobs:
+  python-macos:
+    name: Build macos py${{ matrix.python-version }}
+    runs-on: macos-latest
+    strategy:
+      matrix:
+        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10"]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Build wheels - x86_64
+        uses: messense/maturin-action@v1
+        with:
+          target: x86_64
+          args: -i python${{ matrix.python-version }} --release --out dist ${{ matrix.python-version != '3.9' && '--no-sdist' || '' }}
+      - name: Build wheels - universal2
+        uses: messense/maturin-action@v1
+        env:
+          PYO3_CROSS_LIB_DIR: /Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.8/lib
+        with:
+          args: -i python${{ matrix.python-version }} --release --universal2 --out dist --no-sdist
+      - name: Show wheels generated
+        run: ls -lh dist/
+      - name: Upload wheels
+        uses: actions/upload-artifact@v2
+        with:
+          name: wheels
+          path: dist
+
+  python-windows:
+    name: Build windows ${{ matrix.target }} py${{ matrix.python-version }}
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10"]
+        target: [x64, x86]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+          architecture: ${{ matrix.target }}
+      - name: Build wheels
+        uses: messense/maturin-action@v1
+        with:
+          target: ${{ matrix.target }}
+          args: -i python --release --out dist --no-sdist
+      - name: Show wheels generated
+        run: ls -lh dist
+        shell: bash
+      - name: Upload wheels
+        uses: actions/upload-artifact@v2
+        with:
+          name: wheels
+          path: dist
+
+  python-linux:
+    name: Build linux ${{ matrix.target }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
```

Comment:
why don't we need python version here?

Reply:
When setting this argument
https://github.com/dbt-labs/dbt-extractor/blob/b6efb4784e9129b3ca5b2f1ea4478c1708de1d2b/.github/workflows/release.yml#L84
The `messense/maturin-action@v1` action builds wheels in a docker container and uses the python versions that are available in the docker container. You can see what python versions are supported here https://github.com/messense/maturin-action/blob/51478586be5dfb16a569cb6bbec182fba9f13f79/src/index.ts#L245

I'll add a comment to the source code calling this out!